Update ligeiro.json

warleysantosbr opened the pull request:
#eumeimporto, mas sou afobado 🤣

warleysantosbr committed:
Update ligeiro.json

EndersonMarques commented:
Não faço parte da equipe do ligeiro, nem conheço ele, mas acho que a ideia inicial deveria ser

1. Criar um nove arquivo `json` dentro da pasta [./pix-keys](https://github.com/umpordez/evento-abril/tree/main/pix-keys)
2. Esse arquivo tem que ter um identificador, então pode ser seu nome ou user do github talvez (evita ter duplicidade colisões de json com de pessoas com o mesmo nome): ex: `warleysantosbr.json`
3. E ai sim colocar o conteúdo de sua chave pix lá

O que você fez nesse caso foi substituir o arquivo [ligeiro.json](https://github.com/umpordez/evento-abril/blob/main/pix-keys/ligeiro.json) que já existe no repositório

segue meu PR: https://github.com/umpordez/evento-abril/pull/4

Se você quer usar chave aleatória o `pixAddressKeyType` deve ser  `EVP` `Endereço Virtual de Pagamento` ou no nosso caso é chave aleatória
Está descrito na documentação https://docs.asaas.com/reference/transferir-para-conta-de-outra-instituicao-ou-chave-pix

<img width="637" alt="image" src="https://github.com/user-attachments/assets/ff1d572b-d812-4c02-9825-1b2da4c923d3" />


Outro ponto é que JSON por default não suporta comentários: https://stackoverflow.com/questions/244777/can-comments-be-used-in-json

warleysantosbr committed:
CORREÇÕES

Fazendo as correções que o https://github.com/EndersonMarques me orientou nos comentários. valeuu man

warleysantosbr commented:
@EndersonMarques valeu demais pelas orientações! 🙌

Já corrigi tudo:
✅ Criei meu próprio arquivo JSON
✅ Removi o comentário
✅ Ajustei a chave aleatória pra "EVP"
✅ E não sobrescrevi o ligeiro.json

Obrigado por ter tirado um tempinho pra explicar tudo detalhado, ajudou demais! 👏
Tamo junto! 🚀

Deividy reviewed (changes requested):
NICEE but seems that we are missing a pull/rebase :/ SAD but keep going!!!
